linear_algebra-1.ipynb

```diff
@@ -294,7 +294,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Every subset $R$ of $AxB$ is called relation. We write $aRb$ iff $(a, b) \\in R$"
+    "Every subset $R$ of $AxB$ is called relation. We write $aRb$ if $(a, b) \\in R$"
```

Comment:
Maybe @levants meant here if and only if?

Reply:
@GiorgiBeriashvili consulted w/ @levants 
`iff` means `if and only if`, can you please revert this line?

Reply:
Of course!

I'm on it.